ansible/host_vars/localhost

```diff
@@ -0,0 +1,16 @@
+---
+# FIXME(w-miller): set this more cleverly?
+base_path: "~/tenks/"
+allocations_file_path: "{{ base_path + '/allocations.yml' }}"
+
+flavours:
+  flav0:
+    memory_mb: 4096
+    vcpus: 4
+    volumes:
+      - capacity: 10GB
+    physical_networks:
```

Comment:
To create a multi-homed test node, would it simply be a question of defining additional physical networks here?

Reply:
Yes. For each physical network listed here, a VNIC will be added to the created VM, which will be plugged into the device specified for that physical network in `physnet_mappings`.